index.js

```diff
@@ -4,11 +4,79 @@ const graphqlHTTP = require('express-graphql');
 const cors = require('cors');
 const root = require('./src/resolvers/RootResolver');
 const schema = require('./src/schema/schema');
+const GitHubStrategy = require('passport-github2').Strategy;
+const passport = require('passport');
+const partials = require('express-partials');
+// const util = require('util');
+const session = require('express-session');
+const methodOverride = require('method-override');
 
 const app = express();
+app.use(partials());
+app.use(methodOverride());
+
 const PORT = process.env.PORT || 8000;
+const GITHUB_CLIENT_ID = process.env.GITHUB_CLIENT_ID;
+const GITHUB_CLIENT_SECRET = process.env.GITHUB_CLIENT_SECRET;
+
+const allowCrossDomain = (req, res, next) => {
```

Comment:
@Gwillison415 if you wouldn't mind, I think the `cors` middleware removes the need for this. Try commenting it out and making sure we don't have the CORS issues, and if not needed, please remove. Thanks!